libraries/recipe_helpers.rb

```diff
@@ -0,0 +1,24 @@
+class Chef
+  # Helper class for Datadog Chef recipes
+  class Datadog
+    class << self
+      def api_key(node)
+        run_state_or_attribute(node, 'api_key')
+      end
+
+      def application_key(node)
+        run_state_or_attribute(node, 'application_key')
+      end
+
+      private
+
+      def run_state_or_attribute(node, attr)
```

Comment:
Should we rename `attr`? It's already used by Ruby (not in this context though) - http://ruby-doc.org/core-2.0.0/Module.html#method-i-attr